Formula/a/arttime.rb

```diff
@@ -31,6 +31,16 @@ def install
     system "./install.sh", "--noupdaterc", "--prefix", prefix, "--zcompdir", zsh_completion
   end
 
+  def caveats
+    on_macos do
+      <<~EOS
+        Notification settings on macOS are not fully in control of an application.
+        To check if you have desired notification settings, open following link:
+          https://github.com/poetaman/arttime/issues/11
+      EOS
```

Comment:
Is this specific to Homebrew? Otherwise it should not be a caveat.

Reply:
It is specific to macOS, and homebrew is the only macOS package system to have it.

Reply:
Unless it's specific to the way Homebrew installs things, it should not be a caveat.

Reply:
If someone git clones arttime and runs ``./install.sh``, they get this warning if their OS is macOS. Homebrew's ``brew install arttime`` doesn't print that note.

Notifications are quite central to a timer app like arttime. My concern is that a user installing it from homebrew, would think it's a bug in arttime when they try setting a timer for the first time. The README.md page does mention this caveat (notifications from scripts need to be enabled by the user), but my fear is that many users will miss or give up on during such search... first impression is last impression. If this doesn't fit the definition of caveat, we need something different or need to rethink its definition.

Reply:
> Why is it that daemons are allowed to print that daemon isn't yet started and user needs to take some action? 

Because that is specific to the Homebrew install. It's really that simple.

Caveats are generally ignored by people, so we really try and push back on new ones that don't allign with the requirements for a caveat. I think if you need to add this in many places in macOS it might be better to just print that warning in the app using something like: https://apple.stackexchange.com/questions/460291/how-to-check-whether-notifications-are-enabled-or-not-for-my-app-command-line-on

Reply:
@SMillerDev Our responsibility is to print caveats, user's responsibility is to read it (especially for a new application they are installing for first time). I understand that is the conventional wisdom of definition of homebrew caveats. I fiddled with the link you sent me for a while, it fails to detect if notifications enabled for ``osascript``. And the suggested flags don't work with bundle-id of ScriptEditor. Moreover, an application that just runs ``osascript`` should NOT change users settings. Especially if flags are magic numbers ``41951246 ``. Perhaps Apple would change such numbers over OS upgrades. All of this complication can be avoided if we make room for requests where there is a compelling reason to use caveats. At the end of the day it's a lose-lose situation for both developers and users.

Reply:
`caveats` are not `notices`: they are `caveats` with how Homebrew installs things: "Because Homebrew is X, you must Y" not "Because Arttime is X, you must Y".

Reply:
> @MikeMcQuaid You didn't read my full write up.

@poetaman I read every word you wrote and I don't appreciate you stating otherwise.

> Add another feature to add "further steps" or "installation note".

No.

> We are here to collaborate and help each other.

You are not currently being helpful.

> If you run Homebrew with that mindset, it will be of benefit to many, or if not you will leave a bad legacy.

I've been running Homebrew for 15 years now. Seems to be doing alright. Not sure why I should be taking advice from you, frankly.

Reply:
It doesn't matter what you have done in past 15 years to arttime's users who would open arttime installed from homebrew thinking it works but it doesn't. That's what not being helpful means. Please remove arttime from homebrew till there is a way for users to know that they need to take further steps (change notification settings) before it works.

Reply:
> It doesn't matter what you have done in past 15 years

Kinda does if you're going to be telling me that I need to change my mindset on running this project and talking about my legacy.

> Please remove arttime from homebrew till there is a way for users to know that they need to take further steps

No, sorry. We do not remove software after it's been added to Homebrew on the author's request if it's still being used by our users.

What is in your control: output this message in your software instead and we'll accept a PR to update to that version.

Reply:
Really? I am giving you a rational reason to remove it. You really think that you are serving those homebrew uniformed homebrew users who would install arttime and feel sad that notifications don't work till they take steps? This a slavery mindset, just remove my software? I don't want it to be there on homebrew, either till there is a way to notify users of further steps. It's not serving anyone. If prior homebrew arttime users complaint, feel free to tell them it was developers choice? I mean what reasonably minded platform will force developers to do thing they don't want to?

Reply:
This conversation has run its course, and will not be productive going forward. I am going to close and lock this PR.